tilelang/language/__init__.py

```
@@ -11,7 +11,7 @@
 from . import overrides as _overrides  # noqa: F401
 
 # from .tir import prim_func, macro,  # noqa: F401
-from .v2 import *  # noqa: F401
+from .eager import *  # noqa: F401
```

Ruff (0.14.10)

`from .eager import *` used; unable to detect undefined names

(F403)